ui/selectors/selectors.js

```diff
@@ -376,6 +376,10 @@ export function getIsTestnet(state) {
   return TEST_CHAINS.includes(chainId);
 }
 
+export function getIsNonStandardEthChain(state) {
+  return !(getIsMainnet(state) || getIsTestnet(state));
```

Comment:
you should probably return false if IN_TEST to match some of our other logic to treat ganache as mainnet in tests.